package.json

```
@@ -84,6 +84,7 @@
     "@types/styled-components": "^5.1.4",
     "@xstate/react": "0.8.1",
     "amazon-s3-uri": "^0.0.3",
+    "animejs": "^3.2.1",
```

we should probably lock the package version
```suggestion
    "animejs": "3.2.1",
```